main.py

```diff
@@ -348,7 +357,7 @@ def main() -> int:
                 f.write(src)
     elif args.command == 'sync':
         for img in IMAGES:
-            dest_img = f'getsentry/image-mirror-{img.source.replace("/", "-")}'
+            dest_img = f"getsentry/image-mirror-{img.source.replace('/', '-')}"
 
```

Comment:
unrelated to your patch (same below)

Reply:
This is the pre-commit hook running.

Reply:
no it's not.  black is not configured here